cmd/graphql_playground/cmd/ingest.go

```diff
@@ -176,15 +177,90 @@ func ingestOccurrence(ctx context.Context, client graphql.Client) {
 		Name:      "github.com/guacsec/guac",
 		Tag:       &selectedTag,
 	}
-	occurrence = model.IsOccurrenceSpecInputSpec{
+	occurrence = model.IsOccurrenceInputSpec{
 		Justification: "this artifact is an occurrence of this source",
 		Origin:        "Demo ingestion",
 		Collector:     "Demo ingestion",
 	}
-	respSrc, err := model.IsOccurrenceSrc(context.Background(), client, &src,
+	respSrc, err := model.IsOccurrenceSrc(context.Background(), client, src,
 		model.ArtifactInputSpec{Digest: "6bbb0da1891646e58eb3e6a63af3a6fc3c8eb5a0d44824cba581d2e14a0450cf", Algorithm: "sha256"}, occurrence)
 	if err != nil {
 		logger.Errorf("Error in ingesting: %v\n", err)
 	}
 	fmt.Printf("Response is |%v|\n", respSrc)
 }
+
+func IngestVulnerability(ctx context.Context, client graphql.Client) {
+	logger := logging.FromContext(ctx)
+
+	// test with pkgVersion
+	ns := "openssl.org"
+	version := "3.0.3"
+	pkg := model.PkgInputSpec{
+		Type:       "conan",
+		Namespace:  &ns,
+		Name:       "openssl",
+		Version:    &version,
+		Qualifiers: []model.PackageQualifierInputSpec{{Key: "user", Value: "bincrafters"}, {Key: "channel", Value: "stable"}},
+	}
+	certifyVuln := model.CertifyVulnInputSpec{
+		TimeScanned:    time.Now(),
```

Comment:
getting the following error while ingesting into neo4j:
```
{"level":"error","ts":1677966566.181216,"caller":"cmd/ingest.go:263","msg":"Error in ingesting: input: ingestVulnerability Neo4jError: Neo.ClientError.Statement.TypeError (Unable to construct ZonedDateTime value: `Unknown time-zone ID: Local`)\n\n","stacktrace":"github.com/guacsec/guac/cmd/graphql_playground/cmd.IngestVulnerability\n\t/Users/parth/Documents/pxp928/artifact-ff/cmd/graphql_playground/cmd/ingest.go:263\ngithub.com/guacsec/guac/cmd/graphql_playground/cmd.ingestData\n\t/Users/parth/Documents/pxp928/artifact-ff/cmd/graphql_playground/cmd/ingest.go:45"}
```
What was the solution to this?
Keeping this comment to block merge!

Reply:
this wasn't the same error as the one i encountered - that was more for the missing values, this seems like maybe a formatting issue, maybe on the neo4j ingestion side setting the output of the value encoded to RFC3339 format maybe?

Reply:
I have encountered the same error, it's because the backend expects a different format for the time field.

I'm working on that

Reply:
Keeping this open to make sure all the ingestion and query occur properly before merging.

Reply:
Fixing in #538 , the issue was actually in a mismatch between Go's default timezone in `time.Now()` and neo4j's custom type for local timezones. Switching to UTC solves this and also the fact that our times need to be localized with some timezone.

Reply:
fixed.